packages/tap-runner/schema/tap-runner-options.json

```diff
@@ -0,0 +1,22 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "type": "object",
+  "additionalProperties": false,
+  "properties": {
+    "tap": {
+      "description": "Configuration for @stryker-mutator/tap-runner",
+      "title": "TapRunnerOptions",
+      "additionalProperties": false,
+      "type": "object",
+      "default": {},
+      "properties": {
+        "testFiles": {
+          "description": "Specify glob expressions to test files to run. Defaults to {**/@(test|tests|__test__|__tests__)/**,**/*.@(test|tests|spec)}.@(cjs|mjs|js|jsx|ts|tsx)",
+          "type": "string",
+          "default": "{**/@(test|tests|__test__|__tests__)/**,**/*.@(test|tests|spec)}.@(cjs|mjs|js|jsx|ts|tsx)"
```

Comment:
Now that I think about it, it might be better to allow an array of strings and allow excluding using `!`. But fine to leave it like this for now.